### PR TITLE
Japan

### PR DESCRIPTION
**Pull request by Buraillc**
Updating Japan version of example, I may need more accurate data / continue to improve 🙇🏻‍♀️.

### CODE REVIEW

**Comment by jawalonoski** [comment]
I think at a minimum, we'd want to replace the United Kingdom examples with Japanese examples, perhaps something from Tokyo.

**Comment by Buraillc** [comment]
thanks for getting back, i am working on it and will keep u updated,

**Comment by Buraillc** [comment]
Hi, did it.

File: jp/src/main/resources/geography/demographics_jp.csv [hunk 1]
```diff
@@ -0,0 +1,2 @@
+ID,COUNTY,NAME,STNAME,POPESTIMATE2015,CTYNAME,TOT_POP,TOT_MALE,TOT_FEMALE,WHITE,HISPANIC,BLACK,ASIAN,NATIVE,OTHER,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,00..10,10..15,15..25,25..35,35..50,50..75,75..100,100..150,150..200,200..999,LESS_THAN_HS,HS_DEGREE,SOME_COLLEGE,BS_DEGREE
+1,1,Shrewsbury,West Midlands,2620,Shropshire,486300,0.489,0.511,0.985,0,0.001,0.004,0,0,0.063,0.063,0.063,0.052,0.052,0.052,0.052,0.052,0.076,0.076,0.076,0.076,0.076,0.035,0.035,0.035,0.035,0.035,0.133333333,0.133333333,0.133333333,0.147225,0.147225,0.147225,0.147225,0.1,0.01,0.001,0.18,0.387,22.35,22.35
```
Comment (jawalonoski) [inline jp/src/main/resources/geography/demographics_jp.csv]: You need to change the demographics file too. You should replace Shrewsbury with Tokyo.

File: jp/src/main/resources/providers/hospitals_jp.csv [hunk 2]
```diff
@@ -0,0 +1,9 @@
+,id,name,address,city,state,zip,county,phone,type,ownership,emergency,quality,LAT,LON
+0,10001,Tokyo Medical Hospital,"2-5-1 Higashigaoka, Meguro-ku",Minato,Tokyo,100-8602,Tokyo,03-3411-0111,National Hospital Organization,National Hospital Organization,Yes,5,35.6895,139.69171
+,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,
+,,,,,,,,,,,,,139.69171,
```
Comment (jawalonoski) [inline jp/src/main/resources/providers/hospitals_jp.csv]: You should delete these extra rows with null values, they'll probably cause errors during runtime.

File: jp/test/main/resources/test.properties [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+test.bar = 42
+
+# another comment
+foo = string value
+
+test_state.default = West Midlands
+test_state.alternative = West Midlands
+
+test_town.default = Shrewsbury
+test_town.alternative = Shrewsbury
```
Comment (jawalonoski) [inline jp/test/main/resources/test.properties]: I would recommend updating these so the unit tests use Tokyo.

**Review by jawalonoski — COMMENTED** [review]
Almost there...

File: jp/test/main/resources/test.properties (hunk 3 above)
Comment (Buraillc) [reply jp/test/main/resources/test.properties]: Updated. thx

File: jp/src/main/resources/geography/demographics_jp.csv (hunk 1 above)
Comment (Buraillc) [reply jp/src/main/resources/geography/demographics_jp.csv]: updated. thanks

**Comment by Buraillc** [comment]
will send  revised one over later

**Pull request closed without merging**